support-frontend/assets/helpers/urls/externalLinks.ts

```diff
@@ -32,8 +32,7 @@ const homeDeliveryUrl = `https://www.${getBaseDomain()}/help/2017/dec/11/help-wi
 const defaultIntCmp = 'gdnwb_copts_bundles_landing_default';
 const androidAppUrl =
 	'https://play.google.com/store/apps/details?id=com.guardian';
-const androidFeastAppUrl =
-	'https://play.google.com/store/apps/details?id=uk.co.guardian.feast';
+const feastAppUrl = `https://www.theguardian.com/help/insideguardian/2024/apr/17/introducing-the-feast-app`;
```

Comment:
Shouldn't this be: [https://guardian-feast.go.link?adj_t=1dufrlhf&adj_fallback=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app&adj_redirect_macos=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app](https://guardian-feast.go.link/?adj_t=1dufrlhf&adj_fallback=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app&adj_redirect_macos=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app)

Reply:
Hmmm, I thought what it resolved too was more readable (in a format we had in code-base).  Looking again now though without the `guardian-feast.go.link?` tag at the front am guessing the tracking will fail? To be on safe-side will take this string as is. 

Not sure why it has specific dates `2024/apr/17` in the path either (have asked about this direct).
 
Not sure whether we should use a link without tracking for dev/code too (perhaps without `guardian-feast.go.link?`)?

Reply:
Yep, used full path as suggested.  Done.